### PR TITLE
411 refactor naming convention of member and local variables in the fleet class

### PR DESCRIPTION
**Pull request by Bai-Li-NOAA**
# What is the feature?

Refactor member names in the fleet class

# How have you implemented the solution?

- Use "_m" as a postfix for members of a class
- Use "fleet_" as a prefix for members from fleet class
- Use "interface_" as a prefix for members from interface class

# Does the PR impact any other area of the project?

# How to test this change

Passed GHA workflows can be found [here](https://github.com/NOAA-FIMS/FIMS/actions?query=branch%3A411-refactor-revisit-naming-convention-of-member-and-local-variables-in-a-class)

# Developer pre-PR checklist

- [x] I relied on GitHub actions to :test_tube: things for me while I sat on the :couch_and_lamp:.

### CODE REVIEW

**Comment by ChristineStawitz-NOAA** [comment]
- [ ] add tests on this into the fleet interface test?

File: inst/include/common/information.hpp [hunk 1]
```diff
@@ -174,10 +174,10 @@ class Information {
       FIMS_LOG << "Checking for available fleet index data objects."
                << std::endl;
       // set index data
-      if (f->observed_index_data_id != -999) {
-        uint32_t index_id = static_cast<uint32_t>(f->observed_index_data_id);
-        data_iterator it = this->data_objects.find(index_id);
-        FIMS_LOG << "Input fleet index id = " << index_id << "." << std::endl;
+      if (f->fleet_observed_index_data_id_m != -999) {
+        uint32_t index_id_m = static_cast<uint32_t>(f->fleet_observed_index_data_id_m);
```
Comment (ChristineStawitz-NOAA) [inline inst/include/common/information.hpp:178]: [quibble]: adding `info_` before `index_id_m` would be more consistent with the `fleet_` preface

File: inst/include/common/information.hpp [hunk 2]
```diff
@@ -200,11 +200,11 @@ class Information {
       FIMS_LOG << "Checking for available fleet age comp data objects."
                << std::endl;
       // set age composition data
-      if (f->observed_agecomp_data_id != -999) {
-        uint32_t agecomp_id =
-            static_cast<uint32_t>(f->observed_agecomp_data_id);
-        data_iterator it = this->data_objects.find(agecomp_id);
-        FIMS_LOG << "Input fleet age comp id = " << agecomp_id << "."
+      if (f->fleet_observed_agecomp_data_id_m != -999) {
+        uint32_t agecomp_id_m =
```
Comment (ChristineStawitz-NOAA) [inline inst/include/common/information.hpp:204]: [quibble] : add `info_` prior to `agecomp_id_m`?

**Review by ChristineStawitz-NOAA — APPROVED** [review]

**Comment by ChristineStawitz-NOAA** [comment]
We should add documentation somewhere in the code, and also in the collaborative workflow doc, of the use of `_m` so it's clear to users and developers why we are using the notation.

File: inst/include/common/information.hpp (hunk 1 above)
Comment (Bai-Li-NOAA) [reply inst/include/common/information.hpp:178]: added info_ before index_id_m

File: inst/include/common/information.hpp (hunk 2 above)
Comment (Bai-Li-NOAA) [reply inst/include/common/information.hpp:204]: added info_ before agecomp_id_m

**Comment by Bai-Li-NOAA** [comment]
@ChristineStawitz-NOAA Documentation has been added in the code and an issue has been submitted to the collaborative workflow repo.

**Pull request merged**